timemachine/cpp/src/neighborlist.hpp

```diff
@@ -68,16 +68,17 @@ template <typename RealType> class Neighborlist {
     unsigned int *get_ixn_count() { return d_ixn_count_; }
 
     // get max number of row blocks
-    int B() const { return ceil_divide(this->compute_full_matrix() ? NR_ : NC_, 32); }
+    int B() const;
```

Comment:
I moved the header implementations into the `cu` file so they could access `warp_size` and `tile_size` without needing to add more includes here

Reply:
Actually, would it make sense for `B` and `compute_full_matrix`, `column_blocks`, etc. to be member constants rather than member functions? (since they are deterministic functions of other constants)